src/content/blog/2024/04/25/react-19.md

```diff
@@ -186,7 +186,7 @@ Actions are also integrated with React 19's new `<form>` features for `react-dom
 
 When a `<form>` Action succeeds, React will automatically reset the form for uncontrolled components. If you need to reset the `<form>` manually, you can call the new `requestFormReset` React DOM API.
 
-For more information, see the `react-dom` docs for [`<form>`](/reference/react-dom/components/form), [`<input>`](/reference/react-dom/components/input), and [`<button>`](/reference/react-dom/components/button).
+For more information, see the `react-dom` docs for [`<form>`](/reference/react-dom/components/form), [`<input>`](/reference/react-dom/components/input), and `<button>`.
```

Comment:
We should no longer mention `button` here since it doesn't have react-dom docs.

Reply:
Was wondering if we would have doc written shortly thats why left it as such for now. Can remove if we aren't writing one.

Reply:
Yeah agreed, let's leave it and add docs for `<button>`